exchange/exchange.go

```diff
@@ -248,6 +248,10 @@ func (e *exchange) HoldAuction(ctx context.Context, r AuctionRequest, debugLog *
 
 	e.me.RecordRequestPrivacy(privacyLabels)
 
+	if len(r.StoredAuctionResponses) > 0 || len(r.StoredBidResponses) > 0 {
+		e.me.RecordStoredResponse(r.PubID)
+	}
+
```

Comment:
this is funny, because this is what I did initially and then modified to what it is now.
My only argument is that i prefer to keep it all in one place rather than spread it in code. 
I'm ok with any approaches, what do you think?

Reply:
I'm ok with it.